src/index.js

```diff
@@ -43,6 +43,13 @@ export default function eslint(options = {}) {
                 err.name = 'ESLintError';
                 throw err;
             }
+
+            if (options.breakOnWarning) {
+                const err = Error(' ');
+                err.name = ' ';
```

Comment:
@TrySound my motive here was, to show nothing in the console other than the warnings and errors from eslint. If you think it is not clean, then we could still return some meaningful text like probably, 
name => 'ESLint', 
message => 'Warnings or errors were found'. (same as what you had earlier)

Do you think the text makes sense, or propose what would be nice, as this will show up in the console, for eg with the above
```
Jibys-MacBook-Pro:web-tracker jibyjose$ npm run build

> web-tracker@0.0.1 build /Users/jibyjose/Workspace/Qumram/web-tracker
> rm -rf build; flow check && rollup -c .rolluprc.js;true


Found 0 errors

/Users/jibyjose/Workspace/Qumram/web-tracker/lib/logger.js
  5:3  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)

Warnings or errors were found
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki
```